reference/changelog.md

```diff
@@ -76,6 +81,42 @@ redirect_from: /install/changelog.html
     </tbody>
 </table>
 
+## v0.16.1 {#v161}https://github.com/pulumi/pulumi/pull/2069
+
+Released on October 15, 2018
```

Comment:
is this really the right date?

Reply:
No, it is not.  Copy paste strikes again.